components/app/header/Header.tsx

```diff
@@ -157,7 +159,9 @@ export const Header: React.FC<React.PropsWithChildren<HeaderProps>> = ({
       <div className={clsx(styles.header, styles.flexCenter, rightElement && styles.justifyBetween, className)}>
         <div className={clsx(styles.flexCenter, styles.headerHeight)}>
           <div>
-            <img src={Logo} alt="SubQuery Logo" width={140} />
+            <a href={logoLink ? logoLink : '/'}>
```

Comment:
Please consider to take advantage of typescript 
```suggestion
            <a href={logoLink ?? '/'} target="_blank">
```